src/rust/Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "r-polars"
-version = "0.34.0"
+version = "0.35.0"
```

Comment:
For now, I think 0.34.1 is better because it matches the rust-polars version and is not a breaking change.

Reply:
Done